src/chart_types/xy_chart/state/chart_state.a11y.test.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Store } from 'redux';
+
+import { MockGlobalSpec, MockSeriesSpec } from '../../../mocks/specs';
+import { MockStore } from '../../../mocks/store/store';
+import { GlobalChartState } from '../../../state/chart_state';
+import { getSettingsSpecSelector } from '../../../state/selectors/get_settings_specs';
+
+describe('custom description for screen readers', () => {
+  let store: Store<GlobalChartState>;
+  beforeEach(() => {
+    store = MockStore.default();
+    MockStore.addSpecs(
+      [
+        MockSeriesSpec.bar({
+          data: [
+            { x: 1, y: 10 },
+            { x: 2, y: 5 },
+          ],
+        }),
+      ],
+      store,
+    );
+    MockStore.addSpecs([MockGlobalSpec.settings()], store);
```

Comment:
you can add the settings at the same time of the bar series:
```suggestion
    MockStore.addSpecs(
      [
        MockSeriesSpec.bar({
          data: [
            { x: 1, y: 10 },
            { x: 2, y: 5 },
          ],
        }),
        MockGlobalSpec.settings(),
      ],
      store,
    );
```

Reply:
🙌 75df86b